tests/e2e/test_e2e_slippage_accounting.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+from unittest.mock import MagicMock, Mock
+from datetime import datetime
+
+
+from src.dune_analytics import DuneAnalytics, QueryParameter
+from src.fetch.transfer_file import Transfer
+from src.models import Address, InternalTokenTransfer, Network, SlippagePerSolver
+
+
+def calculate_slippage_for_token(
+    token: Address,
+    internal_trade_list: list[InternalTokenTransfer],
+) -> int:
+    slippage = 0
+    for a in internal_trade_list:
+        if a.from_token_address == token:
+            slippage += a.from_amount
+    return slippage
+
+
+def get_slippage_accounting(
+        dune: DuneAnalytics,
+        period_start: datetime,
+        period_end: datetime
+) -> list[InternalTokenTransfer]:
+    query_str = dune.open_query("./queries/slippage/internal_token_transfers_for_settlements.sql") + \
+        dune.open_query(
+            "./queries/slippage/evaluate_slippage_from_internal_token_transfers.sql")
+    data_set = dune.query_initiate_execute_await(
+        query_str,
+        network=Network.MAINNET,
+        parameters=[
+            QueryParameter.date_type("StartTime", period_start),
+            QueryParameter.date_type("EndTime", period_end),
+            QueryParameter.text_type('TxHash', '0x'),
+        ])
+    return [
+        SlippagePerSolver(
+            solver=row['solver_address'],
+            eth_slippage_wei=row['eth_slippage_wei'],
+        )
+        for row in data_set
+    ]
+
+
+class TestDuneAnalytics(unittest.TestCase):
+    def test_no_solver_has_huge_slippage_values(self):
+        '''
+            This test makes sure that no solver had big slippage values - bigger than 2 ETH -
+            High slippage values are an indication that sometime significant is missing
```

Comment:
While I am not certain that I am convinced by this statement. Why can't slippage be greater than 2 ETH? Anyway, I made a suggestion to the wording.
```suggestion
            This test makes sure that no solver had big slippage (bigger than 2 ETH).
            High slippage indicates that something significant is missing.
```